app.js

```diff
@@ -1,474 +1,570 @@
-var token = ''
+var token = "";
```

Comment:
I still want to keep the `'` rather than moving to `"`. If you can please revert those changes, I'd appreciate it